src/libs/ReportUtils.js

```diff
@@ -809,7 +803,7 @@ function getIOUReportActionMessage(type, total, participants, comment, currency,
  *
  * @returns {Object}
  */
-function buildOptimisticIOUReportAction(sequenceNumber, type, amount, currency, comment, participants, paymentType = '', iouTransactionID = '', iouReportID = '', isSettlingUp) {
```

Comment:
Drafting this now and finding related use around IOU report action sequence numbers. Are these also good to remove? The doc says they should be unused but wanted to double check. @Puneet-here @roryabraham

Reply:
cc @puneetlath 😄